kiwi/repository/apt.py

```diff
@@ -159,12 +159,10 @@ def add_repo(
         if not components:
             components = 'main'
         with open(list_file, 'w') as repo:
-            if repo_gpgcheck is None:
-                repo_line = 'deb {0}'.format(uri)
+            if repo_gpgcheck is False:
+                repo_line = 'deb [trusted=yes][check-valid-until=no] {0}'.format(uri)
```

Comment:
This is the wrong way to do this.

This is the way to do it:
```
deb [trusted=yes check-valid-until=no] {0}
```

Reply:
Reference: [`sources.list(5)`](https://manpages.debian.org/unstable/apt/sources.list.5.en.html#THE_DEB_AND_DEB-SRC_TYPES:_GENERAL_FORMAT)